test/common/libs/cron.test.js

```diff
@@ -181,4 +181,63 @@ describe('cron utility functions', () => {
       expect(result).to.equal(0);
     });
   });
+
+  describe('getPlanContext', () => {
+    const now = new Date(2022, 5, 1);
+
+    function baseUserData (count, offset, planId) {
+      return {
+        purchased: {
+          plan: {
+            consecutive: {
+              count,
+              offset,
+              gemCapExtra: 25,
+              trinkets: 19,
+            },
+            quantity: 1,
+            extraMonths: 0,
+            gemsBought: 0,
+            owner: '116b4133-8fb7-43f2-b0de-706621a8c9d8',
+            nextBillingDate: null,
+            nextPaymentProcessing: null,
+            planId,
+            customerId: 'group-plan',
+            dateUpdated: '2022-05-10T03:00:00.144+01:00',
+            paymentMethod: 'Group Plan',
+            dateTerminated: null,
+            lastBillingDate: null,
+            dateCreated: '2017-02-10T19:00:00.355+01:00',
+          },
+        },
+      };
+    }
+
+    it('offset 0, next date in 3 months', () => {
+      const user = baseUserData(60, 0, 'group_plan_auto');
+
+      const planContext = getPlanContext(user, now);
+
+      expect(planContext.nextHourglassDate)
+        .to.be.sameMoment('2022-08-10T02:00:00.144Z');
```

Comment:
Does this happen on the renewal date? or should it be at the start of the month because this is handled in `grantEndOfTheMonthPerks`?

hmm ok--so the subscription form does say just the month, and mentions that the user will get the Hourglass within 3 days of the start of the month, but you can still mouse over the month text and get a potentially misleading specific date.

Reply:
this is just to get the "context" - the nextHourglassDate is only used in the UI not in the server / cron method - I just added this property since it was mostly in the same-ish context 

the hover was for me for potential debugging - I can remove this hover text as well

Reply:
ok! yeah, let's just remove any debugging-only code/functionality and it should be good, then!

Reply:
removed the tooltip